### PR TITLE
fix Logo alt-Title + add German translations

### LINKED ISSUE #336: [A11y] Logo alt-Title should display "Back to Homepage"
When looking at the alt-Title of the Logo-Image, it displays hardcoded "Plone-Site"

**To Reproduce**
Hover the Page-Logo or Inspect the Logo with the Browser-Inspector

**Expected behavior**
It should be displaying something like "Back to Homepage" to make it a11y-compliant and Screenreader-accessible

**Screenshots**
image
On Hover

image
On Inspect

### PR DESCRIPTION
**Pull request by Molochem**
Fixes #336 
Fixed the Logo-Alt Title to display "Back to homepage", added fitting Translations. Also added a Lot of missing German Translations to `locales/de/LC_MESSAGES/volto.po` i noticed.

image
image

### CODE REVIEW

**Review by davisagli — CHANGES_REQUESTED** [review]
Based on the discussion in https://github.com/plone/volto/pull/5722, this should either say "Visit home page" or use the title of the site.

**Comment by stevepiercy** [comment]
See [discussion on the reopened issue](https://github.com/plone/volto/issues/5720#issuecomment-1922358578) for a single place to discuss how to proceed for this issue.

**Comment by sneridagh** [comment]
@Molochem needs to be adjusted, right?

**Comment by sneridagh** [comment]
@Molochem please update it with the latests developments in Volto.

**Comment by sneridagh** [comment]
@jonaspiterek could you please take over?

**Review by sneridagh — APPROVED** [review]

File: packages/volto-light-theme/src/components/Logo/Logo.jsx [hunk 1]
```diff
@@ -35,8 +35,8 @@ const Logo = () => {
             ? flattenToAppURL(site['plone.site_logo'])
             : LogoImage
         }
-        alt={intl.formatMessage(messages.plonesite)}
-        title={intl.formatMessage(messages.plonesite)}
+        alt={intl.formatMessage(messages.homepage)}
+        title={intl.formatMessage(messages.homepage)}
```
Comment (davisagli) [inline packages/volto-light-theme/src/components/Logo/Logo.jsx:39]: @sneridagh @Molochem This approach is not correct based on the discussion in https://github.com/plone/volto/issues/5720#issuecomment-1956657437

- Neither the link nor the img should have `title`
- The link should have `aria-label="Home"`
- The img should have `alt="Logo"`

**Pull request merged**